readme.md

```diff
@@ -135,6 +135,13 @@ Please see the list below:
 |                  ternary operator                  |    :question:     |
 |                 indexed parameters                 |        :x:        |
 |                      receive                       |    :question:     |
+|         named arguments in function calls          |    :question:     |
```

Comment:
We support things like `f({a: 3, b: 5})`, or is that not what you're referring to here?